openedx/core/djangoapps/zendesk_proxy/v1/views.py

```diff
@@ -0,0 +1,74 @@
+"""
+Define request handlers used by the zendesk_proxy djangoapp
+"""
+from rest_framework import status
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+from rest_framework.throttling import SimpleRateThrottle
+from rest_framework.views import APIView
+
+from openedx.core.djangoapps.zendesk_proxy.utils import create_zendesk_ticket
+
+ZENDESK_REQUESTS_PER_HOUR = 50
```

Comment:
You may need a different rate limiting scheme, I'm not entirely sure. The current v0 implementation limits all users to 50 collective requests per hour.

[Here are the drf docs](http://www.django-rest-framework.org/api-guide/throttling/), the user-specific throttling mentioned there may be more applicable to this usage.